local/nginx/include/redirect_invalid_or_missing_token.conf

```diff
@@ -0,0 +1,10 @@
+if ($missing_token) {
+    # It does not return an error code
+    return 400;
+    break;
```

Comment:
only to check, does this returns 400 if the cookie JWT is missing?

Reply:
I saw the `uniqueLimitCommsServer` and I think 400 redirects there. Please confirm if I understood correctly:
```
location @uniqueLimitCommsServer {
    set_by_lua_block $no_key { return "noKey" }
    limit_req zone=no_token;

    include /etc/nginx/include/comms_proxy.conf;
}
```

Reply:
That's true, as Nginx doesn't allow to put a redirect inside an if the trick that many people use is to define a location using error definition. It's not an error, it doesn't return the 400 status code, but it's the only way that I found to make that work. I really don't like it and left many comments in the code for visibility.